tox.ini

```diff
@@ -26,7 +26,7 @@ application-import-names = splunk-sdk-python
 passenv = LANG
 setenv = SPLUNK_HOME=/opt/splunk
          INPUT_EXAMPLE_UPLOAD=/opt/splunk/var/log/splunk/splunkd_ui_access.log
-whitelist_externals = true
+allowlist_externals = make
```

Comment:
This tells tox that it's OK to run `make`, see: https://tox.readthedocs.io/en/latest/example/basic.html?highlight=make#allowing-non-virtualenv-commands